src/components/GaugeCard/GaugeCard.jsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react';
+import classnames from 'classnames';
+
+import { CARD_CONTENT_PADDING } from '../../constants/LayoutConstants';
+import { CardPropTypes, GaugeCardPropTypes } from '../../constants/PropTypes';
+import Card from '../Card/Card';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+// r value of the circle in SVG
+const radius = 30;
+// radius doubled plus stroke
+const gaugeSize = radius * 2 + 8;
+export const getStrokeDash = (value = 0) => {
+  // circumference of SVG.
+  const circum = 2 * Math.PI * radius;
+  // length of stroke to match percentage
+  return (value * circum) / 100;
+};
+
+export const getColor = (gauge, value) => {
+  let { color } = gauge;
+  let grade = '';
+  const comparisons = {
+    '<': (a, b) => a < b,
+    '>': (a, b) => a > b,
+    '=': (a, b) => a === b,
+    '<=': (a, b) => a <= b,
+    '>=': (a, b) => a >= b,
+  };
+  if (gauge.thresholds) {
+    gauge.thresholds.forEach(thresh => {
```

Comment:
do we need to iterate all of them?  could we use something like `every()` instead?

Reply:
I believe that would still need to iterate over the entire array, no? I think we need to run each comparison to be sure it doesn't trigger .